test/hotspot/jtreg/vmTestbase/nsk/share/locks/DeadlockMaker.java

```diff
@@ -28,7 +28,7 @@
 
 /*
  * Class used to create deadlocked threads. It is possible create 2 or more deadlocked thread, also
- * is is possible to specify resource of which type should lock each deadlocked thread
+ * is possible to specify resource of which type should lock each deadlocked thread
```

Comment:
```suggestion
 * it is possible to specify resource of which type should lock each deadlocked thread
```
It doesn't sound right without _“it”_.